src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignments/test_lmsassignments.py

```diff
@@ -98,11 +98,48 @@ def it_should_run_successfully(test_db_config: ServerConfig, source_system: str)
 
         # assert
         with MSSqlConnection(test_db_config).pyodbc_conn() as connection:
-            LMSAssignment = query(connection, "SELECT * from [lmsx].[Assignment]")
-            assert len(LMSAssignment) == 1
-            assert (
-                int(LMSAssignment[0]["AssignmentIdentifier"]) == 1
-            )  # It is using the identity field from lms.Assignment
+            result = query(connection, "SELECT * from [lmsx].[Assignment]")
+
+        assert len(result) == 1, "There should be one result."
+
+        LMSAssignment = result[0]
+
+        assert (
+            int(LMSAssignment["AssignmentIdentifier"]) == 1
```

Comment:
This is not correct. However, it _is_ what the stored procedure is doing - so here's another bug! This is admittedly confusing; we did a poor job of naming fields. The data type gives it away though:

![image](https://user-images.githubusercontent.com/9324390/127341427-195acb8d-3692-464d-9631-4cbd4c9581df.png)

![image](https://user-images.githubusercontent.com/9324390/127341510-69934803-34f2-4758-a6fb-f36a4f27d9ca.png)

If the Source System were to make a POST request to the API, it would be inserting its _source system identifier_ as the `AssignmentIdentifier`.